src/mono/wasm/debugger/BrowserDebugProxy/MonoProxy.cs

```diff
@@ -213,6 +213,23 @@ protected override async Task<bool> AcceptEvent(SessionId sessionId, JObject par
                         string top_func = args?["callFrames"]?[0]?["functionName"]?.Value<string>();
                         switch (top_func) {
                             // keep function names un-mangled via src\mono\wasm\runtime\rollup.config.js
+                            case "mono_wasm_set_entrypoint_breakpoint":
+                            case "_mono_wasm_set_entrypoint_breakpoint":
+                                {
+                                    var argsNew = JObject.FromObject(new
+                                    {
+                                        callFrameId = args?["callFrames"]?[0]?["callFrameId"]?.Value<string>(),
+                                        expression = "assembly_name_str",
+                                    });
+                                    Result assemblyNameStr = await SendCommand(sessionId, "Debugger.evaluateOnCallFrame", argsNew, token);
+                                    var argsNew2 = JObject.FromObject(new
+                                    {
+                                        callFrameId = args?["callFrames"]?[0]?["callFrameId"]?.Value<string>(),
+                                        expression = "token",
+                                    });
+                                    Result tokenResult = await SendCommand(sessionId, "Debugger.evaluateOnCallFrame", argsNew2, token);
+                                    return await OnSetEntrypointBreakpoint(sessionId, assemblyNameStr.Value["result"]["value"].Value<string>(), tokenResult.Value["result"]["value"].Value<int>() & 0xffffff, token);
```

Comment:
I would like to know: why we change the token we got from debugger here? Can't we use the original one?